patches/api/0474-Add-preliminary-API-for-working-with-new-mob-AI-syst.patch

```diff
@@ -0,0 +1,175 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Janet Blackquill <uhhadd@gmail.com>
+Date: Fri, 26 Apr 2024 23:45:11 -0400
+Subject: [PATCH] Add preliminary API for working with 'new' mob AI system
+
+
+diff --git a/src/main/java/io/papermc/paper/entity/ai/Activities.java b/src/main/java/io/papermc/paper/entity/ai/Activities.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..d03d81256a8ba9a07e1a6d3990f1d3654ea608d2
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/entity/ai/Activities.java
+@@ -0,0 +1,32 @@
++package io.papermc.paper.entity.ai;
++
++import org.jetbrains.annotations.NotNull;
++
++public interface Activities {
++    public @NotNull Activity getAdmireItem();
```

Comment:
See below, use api-generator and key based approach.

Reply:
... how do i use api-generator? i don't see any documentation in the repo

Reply:
fixed